### PR TITLE
Update manifest and changes file for release

### PR DESCRIPTION
**Pull request by jscarty**

### CODE REVIEW

**Comment by pvoglhofer** [comment]
should probably wait until the new Proxy class is merged.

File: Makefile.PL [hunk 1]
```diff
@@ -30,7 +30,7 @@ WriteMakefile(
     },
     PREREQ_PM         => {
         'Gtk3'              => 0,
-        'Gtk3::WebKit2'     => 0.01,
+        'Gtk3::WebKit2'     => 0.011,
```
Comment (pvoglhofer) [inline Makefile.PL:33]: Gtk3::WebKit2 is on 0.012

File: Changes [hunk 2]
```diff
@@ -1,5 +1,8 @@
 Revision history for WWW::WebKit2
 
-0.11 Mon Mar 23 15:27:00 CET 2020
+0.11 Fri July 24 15:27:00 CET 2020
```
Comment (pvoglhofer) [inline Changes:3]: This should be 0.12 now.

**Review by pvoglhofer — CHANGES_REQUESTED** [review]

File: Makefile.PL (hunk 1 above)
Comment (jscarty) [reply Makefile.PL:33]: Gtk3::Webkit2 0.012 only includes text changes. 0.011 had some needed changes for the release (seemed to cause problems building on cpan). It doesn't need the newer version.

**Review by pvoglhofer — APPROVED** [review]

**Pull request merged**